### DISCUSSION TITLE
how can we setup sync to cloud?

### DISCUSSION THREAD

**Discussion by braindevices** [aashishksahu/SafeSpace#35] [Q&A]
with cryfs, we can just sync the encrypted volume as normal file. But I wonder how can we do this with safespace.  In general this is already better than google Files's safe folder, because it support more file types. However, I cannot think of reasonable way to sync the files in safespace, if we do not do extra encryption. Maybe safespace support some E2EE service that I do not know?

**Comment by aashishksahu** [comment]
As of now SafeSpace doesn’t support cloud sync and that is by design as I intended the app to be offline only. Therefore, Cloud syncing is not on the roadmap. Although, a not so convenient way of cloud syncing is exporting a zip backup and using file encryption apps to encrypt and upload the backup to cloud manually. Thanks for using my app though :)
